cmd/ingress-operator/main.go

```diff
@@ -174,6 +176,23 @@ func createDNSProvider(cl client.Client, operatorConfig operatorconfig.Config, d
 			return nil, fmt.Errorf("failed to create GCP DNS provider: %v", err)
 		}
 		dnsProvider = provider
+	case configv1.LibvirtPlatformType:
+		cluster := strings.Split(dnsConfig.Spec.BaseDomain, ".")[0]
+		appDomain := strings.Replace(dnsConfig.Spec.BaseDomain, cluster, "apps", 1)
+		// LIBVIRT_URL must be set as 'qemu+ssh://x.x.x.x/system'
+		libvirtUrl := os.Getenv("LIBVIRT_URI")
```

Comment:
No, you need to get this passed by the Installer to ensure you're using the same connection as Installer and other components. As i said before, have a look at how `cluster-api-provider-libvirt` is doing it.